homeassistant/components/deconz/binary_sensor.py

```diff
@@ -45,7 +45,12 @@ def async_add_sensor(sensors, new=True):
         )
     )
 
-    async_add_sensor(gateway.api.sensors.values())
+    async_add_sensor(
+        [
+            gateway.api.sensors[key]
+            for key in sorted(gateway.api.sensors.keys(), key=int)
```

Comment:
Iterating a dict iterates the keys by default. We can remove `.keys()`.

Reply:
Thanks!